frontend/test/unit/specs/utils/decode-data.spec.js

```diff
@@ -37,7 +37,7 @@ describe("decodeData", () => {
     expect(decodeData(data)).toBe("musée")
   })
 
-  it("shouldnt throw exception", () => {
+  it("shouldn't throw exception", () => {
```

Comment:
Maybe an ESLint rule for test names to start with "should"/"should not"/"does"/"does not"? I don't know if that seems excessively restrictive without much additional benefit.

Reply:
I like it from a perspective of encouraging descriptive (and "self documenting") test names, but it would require a _lot_ of changes in our tests, I think... we've got a lot of tests that aren't formatted that way.